.obelisk/impl/github.json

```diff
@@ -1,7 +1,8 @@
 {
   "owner": "obsidiansystems",
   "repo": "obelisk",
-  "branch": "master",
-  "rev": "7ad33cbe3e84b209e83c505ce25486445bbd602e",
-  "sha256": "0dlk8y6rxc87crw7764zq2py7nqn38lw496ca1y893m9gdq8qdkz"
+  "branch": "cn/ignore-more-libuv-tests-obelisk-side",
```

Comment:
Can we just add this change: https://github.com/obsidiansystems/nixpkgs/commit/c2304d2565502216d65fd68660c39784fe253104 to an overlay somwhere more local (obApp.nix or kpkgs perhaps ?)